sentry-anyhow/src/lib.rs

```diff
@@ -13,6 +26,8 @@
 //!     capture_anyhow(&err);
 //! }
 //! ```
+//!
+//! [`anyhow::Error`]: https://docs.rs/anyhow/*/anyhow/struct.Error.html
```

Comment:
I think intra-doc-links should handle this?

Which reminds me, the relative links from above relative links will also fail when this is converted to the README via cargo-readme :-(

Reply:
I don't understand what happened here.  When trying to use the plain intra-doc links  `cargo +nightly doc` did not give any warning, suggesting it was happy with the link, but they did not work, they just rendered literally.  I have no idea what I was doing wrong and failed to find.  This was the only way I got this to link correctly, it's not very nice but at least it's not wrong.

If you can get it to work with the intra-doc link then let me know what I did wrong!